cdap-ui/app/hydrator/templates/create/toppanel.html

```diff
@@ -93,22 +93,26 @@
       <div class="button-label">Import</div>
     </div>
     <div class="btn"
-         ng-click="HydratorPlusPlusTopPanelCtrl.onExport()">
+         ng-click="!HydratorPlusPlusTopPanelCtrl.hasNodes || HydratorPlusPlusTopPanelCtrl.onExport()"
+         ng-disabled="!HydratorPlusPlusTopPanelCtrl.hasNodes">
       <span class="fa icon-export"></span>
       <div class="button-label">Export</div>
     </div>
     <div class="btn"
-         ng-click="HydratorPlusPlusTopPanelCtrl.onSaveDraft()">
+         ng-click="!HydratorPlusPlusTopPanelCtrl.hasNodes || HydratorPlusPlusTopPanelCtrl.onSaveDraft()"
+         ng-disabled="!HydratorPlusPlusTopPanelCtrl.hasNodes">
       <span class="fa icon-savedraft"></span>
       <div class="button-label">Save</div>
     </div>
     <div class="btn"
-         ng-click="HydratorPlusPlusTopPanelCtrl.onValidate()">
+         ng-click="!HydratorPlusPlusTopPanelCtrl.hasNodes ||HydratorPlusPlusTopPanelCtrl.onValidate()"
```

Comment:
the button click triggers even if it's disabled because its a <div> and not a <button>